reporter/src/main/kotlin/reporters/CycloneDxReporter.kt

```diff
@@ -134,8 +136,9 @@ class CycloneDxReporter : Reporter {
 
                 bom.addExternalReference(ExternalReference.Type.WEBSITE, project.homepageUrl)
 
-                val licenseNames = project.declaredLicensesProcessed.allLicenses +
-                        input.ortResult.getDetectedLicensesForId(project.id, input.packageConfigurationProvider)
+                val licenseNames = input.licenseInfoResolver.resolveLicenseInfo(project.id)
+                    .getLicenseNames(LicenseSource.DECLARED)
```

Comment:
Is this really the same as before, because before we also got the *detected* licenses for the project?

Reply:
Oh right, good catch!